client/src/App.js

```diff
@@ -2,11 +2,13 @@ import React from 'react';
 import logo from './logo.svg';
 import './App.css';
 import Footer from './component/Footer.js'; 
+import NavigationBar from './component/Navigation-Bar.js'; 
```

Comment:
Change file to a `.jsx` file instead of a `.js`
```suggestion
import NavigationBar from './component/Navigation-Bar.jsx'; 
```